ci-operator/config/openshift/release/openshift-release-master__ci-4.18.yaml

```diff
@@ -232,6 +232,14 @@ tests:
       enable:
       - observers-resource-watch
     workflow: openshift-e2e-gcp-ovn
+- as: e2e-gcp-ovn-xpn
+  interval: 168h
+  steps:
+    cluster_profile: gcp-openshift-gce-devel-ci-2
```

Comment:
the cluster profile determines which gcp project this runs in. the original xpn job is using `gcp`, so let's use that here as well:

https://github.com/openshift/release/blob/f4865629e84ebea449f7b98471cc5ba6fee5ea4a/ci-operator/config/openshift/installer/openshift-installer-master.yaml#L615

needs to be updated on all branches